app/models/concerns/purchase/blockable.rb

```diff
@@ -3,6 +3,25 @@
 module Purchase::Blockable
   extend ActiveSupport::Concern
 
+  included do
+    include AttributeBlockable
+
+    attr_blockable :browser_guid
+    attr_blockable :ip_address
+    attr_blockable :email
+    attr_blockable :paypal_email, attribute: :email
```

Comment:
`object_type:` might be clearer to match the field on `BlockedObject` (if I'm understanding what it maps to correctly)